src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -438,7 +438,7 @@ private void LoadPaneChanged()
 					(MainWindow.Instance.Bounds.Width > 450 && MainWindow.Instance.Bounds.Height > 450 || RootGrid.ActualWidth > 700 && MainWindow.Instance.Bounds.Height > 360);
 
 				ViewModel.ShouldPreviewPaneBeDisplayed = (!isHomePage || isMultiPane) && isBigEnough;
-				ViewModel.ShouldPreviewPaneBeActive = UserSettingsService.InfoPaneSettingsService.IsEnabled && ViewModel.ShouldPreviewPaneBeDisplayed;
+				ViewModel.ShouldPreviewPaneBeActive = UserSettingsService.InfoPaneSettingsService.IsInfoPaneEnabled && ViewModel.ShouldPreviewPaneBeDisplayed;
```

Comment:
PreviewPane to InfoPane here?

Reply:
What do you mean?

Reply:
```diff
-ShouldPreviewPaneBeActive & ShouldPreviewPaneBeDisplayed
+ShouldInfoPaneBeActive & ShouldInfoPaneBeDisplayed
```

Reply:
I'd have to check the code but I think you're right.